spec/rubocop/cop/lint/redundant_literal_conversion_spec.rb

```diff
@@ -0,0 +1,266 @@
+        RUBY
+      end
+
+      it 'registers an offense and corrects when calling `to_s` on a `to_s` call with safe navigation' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          foo&.#{conversion}&.#{conversion}
+               _{conversion}  ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo&.#{conversion}
+        RUBY
+      end
+
+      it 'registers an offense and corrects when calling `to_s` on a `to_s` call with an argument' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          foo.#{conversion}(2).#{conversion}
+              _{conversion}    ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo.#{conversion}(2)
+        RUBY
+      end
+
+      it 'registers an offense and corrects when calling `to_s` on a `to_s` call with an argument and safe navigation' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          foo&.#{conversion}(2)&.#{conversion}
+               _{conversion}     ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo&.#{conversion}(2)
+        RUBY
+      end
+
+      it 'registers an offense and corrects when the redundant `to_s` is chained further' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          foo.#{conversion}.#{conversion}.bar
+              _{conversion} ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo.#{conversion}.bar
+        RUBY
+      end
+
+      it 'registers an offense and corrects when the redundant `to_s` is chained further with safe navigation' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          foo&.#{conversion}&.#{conversion}&.bar
+               _{conversion}  ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo&.#{conversion}&.bar
+        RUBY
+      end
+
+      it 'registers an offense for a `to_s` call wrapped in parens' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          (foo.#{conversion}).#{conversion}
+               _{conversion}  ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          (foo.#{conversion})
+        RUBY
+      end
+
+      it 'registers an offense for a `to_s` call wrapped in multiple parens' do
+        expect_offense(<<~RUBY, conversion: conversion)
+          ((foo.#{conversion})).#{conversion}
+               _{conversion}    ^{conversion} Redundant `#{conversion}` detected.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          ((foo.#{conversion}))
+        RUBY
+      end
+    end
+  end
+
+  it 'does not register an offense for chaining different conversion methods' do
+    expect_no_offenses(<<~RUBY)
+      foo.to_i.to_s
+    RUBY
+  end
+
+  context '`to_s`' do
```

Comment:
These cases should be in `describe`, not `context`.

```suggestion
  describe '`to_s`' do
```

Reply:
Thank you, updated.